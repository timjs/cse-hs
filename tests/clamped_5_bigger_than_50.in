5
cior(afrz(fos,efdx(uep,drrd(cxq,gnk(ajya,dmop(brdd,bjfq(anki(dgfx,dmdj),dihy)))))),dyjw(acei,cieo))
mult(mult(minus,b(minus(a,a(baz(a,plus),y(baz(minus(y(minus,plus),f(mult,plus)),y),y))),mult)),pow)
x(x,pow(f(plus(b(y(plus(y,minus),pow),baz),y),b(y(y(plus,x),plus(x,f(x(b,baz),mult))),minus)),baz))
epfu(dvzo,ejho(oyy,tkt(bozv(eohh(ahio(crhy(jag(btch,znt(czdc,ayab)),eayb),bsvv),cjpw),czkv),axex)))
hnx(cvlo,dqcs(aemi,wvl(dqrd(tti(djbh(ecu(epxc,cylt),crfj),dpgp),dook(dshs,boso(bpsp,ehyw))),abjn)))
