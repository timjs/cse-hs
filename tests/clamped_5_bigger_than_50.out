cior(afrz(fos,efdx(uep,drrd(cxq,gnk(ajya,dmop(brdd,bjfq(anki(dgfx,dmdj),dihy)))))),dyjw(acei,cieo))
mult(mult(minus,b(minus(a,a(baz(6,plus),y(baz(minus(y(3,9),f(mult,9)),y),16))),15)),pow)
x(x,pow(f(plus(b(y(plus(y,minus),pow),baz),9),b(y(y(plus,2),plus(2,f(x(b,12),mult))),10)),12))
epfu(dvzo,ejho(oyy,tkt(bozv(eohh(ahio(crhy(jag(btch,znt(czdc,ayab)),eayb),bsvv),cjpw),czkv),axex)))
hnx(cvlo,dqcs(aemi,wvl(dqrd(tti(djbh(ecu(epxc,cylt),crfj),dpgp),dook(dshs,boso(bpsp,ehyw))),abjn)))
